setup.py

```diff
@@ -35,24 +31,14 @@
         author = {Gaetan K. W. Kenway and Graeme J. Kennedy and Joaquim R. R. A. Martins}
       }
       """,
-      long_description_content_type="text/markdown",
-      keywords='spline b-spline optimization',
-      author='',
-      author_email='',
-      url='https://github.com/mdolab/pyspline',
-      license='Apache License Version 2.0',
-      packages=[
-          'pyspline',
-      ],
-      package_data={
-          'pyspline': ['*.so']
-      },
-      install_requires=[
-            'numpy>=1.16',
-            'scipy>=1.2',
-      ],
-      classifiers=[
-        "Operating System :: Linux",
-        "Programming Language :: Python, Fortran"]
-      )
-
+    long_description_content_type="text/markdown",
+    keywords="spline b-spline optimization",
+    author="",
+    author_email="",
+    url="https://github.com/mdolab/pyspline",
+    license="Apache License Version 2.0",
+    packages=["pyspline",],
```

Comment:
The reason they disagreed has to do with the fact that there is a trailing comma. if you delete it then they should agree.